fractions/keycloak-microprofile-jwt/README.adoc

```diff
@@ -0,0 +1,9 @@
+= Keycloak Microprofile JWT
+
+Uses Keycloak Adapter API to validate the tokens issued by Keycloak and makes these tokens accessible with Microprofile JWT API.
+ 
+It depends on the `keycloak.json` configuration file and ignores the Microprofile JWT configuration properties given that the mapping from the former to the latter is not always possible. It does not use the WildFly Keycloak Adapter subsystem.
+
+Standard Microprofile JWT `MP-JWT` authentication type is required by this fraction.
+
+
```

Comment:
Please mention somewhere that, unlike MP JWT, this can redirect to the login page, when you mentioned that, I found it pretty useful for me to understand what this fraction offers.

Reply:
For the redirect to work more work needs to be done to overcome the current microprofile-jwt login module constraints, I will open a new issue after this PR goes in

Reply:
Ah OK. I think you can open a JIRA already, but sure, doesn't need to be done in this PR.